sources/packages/backend/apps/api/src/route-controllers/student-appeal/models/student-appeal.dto.ts

```diff
@@ -23,6 +23,8 @@ export class StudentAppealRequestAPIInDTO {
   @IsDefined()
   @JsonMaxSize(JSON_10KB)
   formData: unknown;
+  @IsDefined()
+  files: string[];
```

Comment:
Is it expected to come as `[ ]` for appeals without files? or undefined?

Reply:
Yes, it is expected to come as `[ ]` for appeals without files because `getAssociatedFiles(appealForm)` returns an array of strings or an empty array (`const getAssociatedFiles: (form: any) => string[]`)